### PR TITLE
[api] public api for simple request xml

### PR DESCRIPTION
**Pull request by adrianschroeter**
used for some openSUSE bot

### CODE REVIEW

**Comment by bgeuken** [comment]
LGTM

File: src/api/app/controllers/public_controller.rb [hunk 1]
```diff
@@ -153,6 +153,13 @@ def distributions
     render 'distributions/index'
   end
 
+  # GET /public/request/:number
+  def show_request
+    required_parameters :number
+    req = BsRequest.find_by_number!(params[:number])
```
Comment (bgeuken) [inline src/api/app/controllers/public_controller.rb]: What happens in case of an exception?

Comment (adrianschroeter) [inline src/api/app/controllers/public_controller.rb]: Everything handled by the default handlers, but I added a test case for this here: ce83ef4ad3ebaa592020468e79a39377b3a98360

Comment (bgeuken) [inline src/api/app/controllers/public_controller.rb]: thanks :+1:

**Pull request merged**